aten/src/ATen/native/BlasKernel.cpp

```diff
@@ -196,15 +198,36 @@ void gemv(char trans, int64_t m, int64_t n, scalar_t alpha, scalar_t *a, int64_t
   } else {
     if (beta != scalar_t(1) && beta != scalar_t(0)) scal<scalar_t>(m, beta, y, incy);
 
+    bool is_low_precision = !std::is_same<opmath_t, scalar_t>::value;
+    std::vector<opmath_t> sum;
+    if (is_low_precision) {
+      sum.reserve(m);
+      for (const auto i : c10::irange(m)) {
+        sum[i] = y[i * incy];
+      }
+    }
```

Comment:
This is illegal, you've can't write to the vector with size 0. It would also make sense to avoid the copy here so we only access `y` once at the end.
```suggestion
    std::vector<opmath_t> sum;
    if (is_low_precision) {
      sum.resize(m);
    }
```

Reply:
Thanks for reviewing, resolved.